Translated/Quests/Story/05 DOLLS on Mt. Magnus 2/qst_1200050032.csv

```diff
@@ -1,3 +1,3 @@
 name010#0,"""Bruder"""
-text010#0,"""大きく体勢を崩したということは、<br>攻撃が確実に効いている証拠だ。"""
-text020#0,"""このまま畳みかけていくぞ。<br>攻撃の手を休めるな！"""
+text010#0,"""If its posture is thrown way off balance, that's when you know your attacks will land with devastating force."""
```

Comment:
Seems something more like
"That we threw it off balance is proof that our attacks are definitely working."